help/C/merge.page

```diff
@@ -9,26 +9,18 @@
 
 <p>
     <app>Time Tracker</app> does its best to avoid overlaps in activities.
-    If you create an activity in the middle of an existing activity, the former
-    will be split into two. In other cases of overlapping, the previous entries
-    will be shrunk.
+    If you create an activity whose start and end time both lie within an
+    existing activity, the existing activity will be split into two. In
+    other cases of overlapping, the previous entries will be shrunk.
 </p>
-<note style="bug">
-    <title>Former activity not split</title>
-    <p>
-        The former activity is not "split into two" actually,
-        but merely shortened to end when the new activity starts.
-        Reported as <link href="https://github.com/projecthamster/hamster/issues/396">issue 396</link>.
-    </p>
-</note>
```

Comment:
This issue is not entirely solved. I would prefer to leave this comment.
Maybe not just "former activity" but "ongoing former activity" to be more precise.